base/cadvisor/cadvisor.ClusterRoleBinding.yaml

```diff
@@ -16,4 +16,4 @@ subjects:
 - kind: ServiceAccount
   name: cadvisor
   # value of namespace needs to match namespace value in base/cadvisor/cadvisor.ServiceAccount.yaml
-  namespace: default
+  namespace: ns-sourcegraph
```

Comment:
I would make this only `sourcegraph` its implicitly an NS.

Reply:
`ns-sourcegraph` is the default: https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/deploy-sourcegraph%24+namespace:+ns-sourcegraph&patternType=structural

Reply:
That is unfortunate.